views/default/js/mentions/autocomplete.js

```diff
@@ -10,6 +10,24 @@ define(function(require) {
 	var $ = require('jquery');
 	var elgg = require('elgg');
 
+	if (require.specified('ckeditor')) {
```

Comment:
Ah, I see this handles tinymce as well. That's a bummer. These should probably be split to separate modules. Oh well...